jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/PropertyNames.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.gradle;
+
+/** Names of system properties used to set configuration via commandline. */
+class PropertyNames {
```

Comment:
These are the same as the Maven ones right? Let's put them in `plugins-common`?

Reply:
Also, what do you think about using an enum for this?

Reply:
I think they could go in `plugins-common`, although I thought we were keeping configuration naming out of `plugins-common` for some reason (maybe I'm misremembering a conversation we had a while ago, might have been something else).

I don't think we should use an enum here since this is specifically a collection of string constants, not a special data type.